facade/src/main/java/com/redhat/lightblue/migrator/facade/TimeoutConfiguration.java

```diff
@@ -118,7 +118,17 @@ public long getMS(String methodName, FacadeOperation op, Type type) {
                 log.debug("{} config not found for bean {} using global timeout", type, beanName);
             }
 
-            timeout = defaultTimeoutMS;
+            switch (type) {
+                case timeout: {
+                    timeout = defaultTimeoutMS; break;
+                }
+                case slowwarning: {
+                    timeout = 2 * defaultTimeoutMS; break;
```

Comment:
I am generally good with this change, but am not sure that I understand the point of the Type enum. I see that it previously existed, but until this it doesn't look like it is used for anything except logging. Why not just let the constructor handle this?